.eslintrc.json

```diff
@@ -19,111 +18,186 @@
     "parser": "@typescript-eslint/parser",
     "plugins": ["react", "react-native", "@typescript-eslint"],
     "root": true,
+    "extends": ["airbnb/legacy"],
```

Comment:
at web sdk our reason to use this was we had to use esm5. here I think 6 would be fine too. rules can stay same but this part I think we can change.